### PR TITLE
Add racket-describe

### PR DESCRIPTION
**Pull request by goderich**
A help mode to look up Racket documentation inside Emacs.

### CODE REVIEW

**Comment by goderich** [comment]
Hmmm... Not sure why checks are failing. How should I diagnose this?

File: modes/racket-describe/evil-collection-racket-describe.el [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;; Bindings for `racket-describe'.
+
+;;; Code:
+(require 'evil-collection)
+(require 'racket-describe)
```
Comment (jojojames) [inline modes/racket-describe/evil-collection-racket-describe.el:31]: I believe it'll be (require 'asdf nil t) or something. You can look at the other files for examples.

In toplevel form:

modes/racket-describe/evil-collection-racket-describe.el:31:1:Error: Cannot open load file: No such file or directory, racket-describe

make: *** [compile] Error 1

**Comment by goderich** [comment]
Thank you @jojojames , I fixed the issue!

**Comment by jojojames** [comment]
Thanks!

**Pull request merged**